src/layer/Popup.js

```diff
@@ -191,7 +191,8 @@ export var Popup = DivOverlay.extend({
 		if (this.options.closeButton) {
 			var closeButton = this._closeButton = DomUtil.create('a', prefix + '-close-button', container);
 			closeButton.href = '#close';
-			closeButton.innerHTML = '&#215;';
+			// This is actually a button labelled "Close" for accessibility
+			closeButton.innerHTML = '<span role="button" aria-label="Close">&#215;</span>';

```

Comment:
I think changing from `aria-label="Close"` to `aria-label="Close popup"` is an improvement, see https://github.com/Leaflet/Leaflet/issues/7399.

Reply:
agreed, but that would only really make sense if you can make sure that there is only one popup visible at a time - otherwise including a title of some sort with the label would be necessary to avoid confusion I think.